spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/DeploymentStateResource.java

```diff
@@ -30,7 +30,7 @@
 public enum DeploymentStateResource {
 
 	DEPLOYING("deploying", "Deploying", "The app or group is being deployed."),
```

Comment:
to handle the first part of the story should we also say, "The stream is being deployed"  If you agree with me I can fix on merge.

Reply:
Sure, I think that looks better. Thanks @cppwfs